vms/platformvm/state/diff.go

```diff
@@ -294,6 +296,26 @@ func (d *diff) SetSubnetOwner(subnetID ids.ID, owner fx.Owner) {
 	d.subnetOwners[subnetID] = owner
 }
 
+func (d *diff) GetSubnetManager(subnetID ids.ID) (ids.ID, []byte, error) {
+	if manager, exists := d.subnetManagers[subnetID]; exists {
+		return manager.ChainID, manager.Addr, nil
+	}
+
+	// If the subnet manager was not assigned in this diff, ask the parent state.
+	parentState, ok := d.stateVersions.GetState(d.parentID)
+	if !ok {
+		return ids.Empty, nil, ErrMissingParentState
+	}
+	return parentState.GetSubnetManager(subnetID)
+}
+
+func (d *diff) SetSubnetManager(subnetID ids.ID, chainID ids.ID, addr []byte) {
+	d.subnetManagers[subnetID] = chainIDAndAddr{
```

Comment:
Where is `subnetManagers` initialized? Test?

Reply:
It wasn't 😓... fixed